code401challenges/Data-Structures/src/main/java/datastructure/Library.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package datastructure;
+
```

Comment:
we only like nested classes like this in very special circumstances, please move everything you have in this nested LinkedList class into the empty LinkedList class you have in your file structure. the Library.java is not needed, it is just a file gradle likes to set up for you.
please also move the nested Node class you have here into its own file/class.

i would really like you to do this